src/http_upload/mod_http_upload.erl

```diff
@@ -258,9 +258,9 @@ token_bytes(HostType) ->
     gen_mod:get_module_opt(HostType, ?MODULE, token_bytes).
 
 
--spec max_file_size(mongooseim:host_type()) -> pos_integer() | undefined.
+-spec max_file_size(mongooseim:host_type()) -> pos_integer() | infinity.
 max_file_size(HostType) ->
-    gen_mod:get_module_opt(HostType, ?MODULE, max_file_size).
+    gen_mod:get_module_opt(HostType, ?MODULE, max_file_size, infinity).
```

Comment:
I'd keep this line unchanged, because the option should always be present, right? This could create confusion for the reader about the actual default.